[Docs] Add note on `--exclude` about possibly verbose regex (#4145)
psf/black: docs/usage_and_configuration/the_basics.md

directories on all platforms (Windows, too). By default, Black also ignores all paths
listed in `.gitignore`. Changing this value will override all default exclusions.

If the regular expression contains newlines, it is treated as a verbose regular

> expression. This is typically useful if setting these options in a `pyproject.toml`
> config file, see [Configuration format](#configuration-format) for more information.

Reviewer: ```suggestion
expression. This is typically useful when setting these options in a `pyproject.toml`
configuration file; see [Configuration format](#configuration-format) for more information.
```

Also, maybe link to https://docs.python.org/3/library/re.html#re.VERBOSE to explain what "verbose" means.
